.azure-devops/templates/nightly-tests.yml

```diff
@@ -32,12 +32,12 @@ steps:
     continueOnError: true
     displayName: 'All integration tests'
     inputs:
-      azureSubscription: az-cli-nightly
+      azureSubscription: az_cli_nightly_conn
       scriptType: bash
       scriptLocation: inlineScript
       inlineScript: |
         export COVERAGE_FILE=.coverage.all
-        pytest -vv azext_iot/tests -k "_int" --cov=azext_iot --cov-config .coveragerc --junitxml=junit/test-iotext-int.xml
+        pytest -vv azext_iot/tests/iothub -k "_int" --dist=loadfile -n 12 --reruns 2 --reruns-delay 60 --cov=azext_iot --cov-config .coveragerc --junitxml=junit/test-iotext-int.xml
```

Comment:
Parallelization threshold should be configurable, not hard-coded.

Reply:
Yeah I am testing with the optimal value for number of hubs, will make this configurable and default to the optimal value